modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -34,7 +34,7 @@ def set_submission_attributes
           raise Common::Exceptions::RecordNotFound, params[:nod_id] unless @appeal
 
           @submission_attributes ||= {
-            source: request.headers['X-Consumer-Username'],
+            source: params['headers']['X-Consumer-Username'],
```

Comment:
shouldn't this be `headers['X-Consumer-Username']` like it is for the HigherLevelReviewsController?

Reply:
hlr's controller has a method for parsing the headers
`  def headers
    HEADERS.reduce({}) do |acc, header_key|
      header_value = request.headers[header_key]

      header_value.nil? ? acc : acc.merge({ header_key => header_value })
    end
  end`
kinda doesn't seem worth it here?